horreum-web/package.json

```diff
@@ -8,6 +8,7 @@
     "@patternfly/patternfly": "5.1.0",
     "@patternfly/react-core": "5.1.1",
     "@patternfly/react-table": "5.1.1",
+    "@types/history": "4.7.11",
```

Comment:
please see comment below about "history": "4.10.1",

Please remove this dependency

Reply:
read above comment